integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -22,11 +22,12 @@ export class OpfResourceLoaderService {
   protected document = inject(DOCUMENT);
   protected platformId = inject(PLATFORM_ID);
 
-  protected readonly OPF_RESOURCE_ATTRIBUTE_KEY = 'data-opf-resource';
   protected readonly CORS_DEFAULT_VALUE = 'anonymous';
+  protected readonly OPF_RESOURCE_LOAD_ONCE_ATTR_KEY = 'opf-load-once';
```

Comment:
Could we use for consistency: `OPF_RESOURCE_LOAD_ONCE_ATTRIBUTE_KEY`?

Reply:
Also would be good in my opinion to prefix all custom html attributes with "data-".
https://www.w3schools.com/tags/att_data-.asp

Can we change attribute key to: `data-opf-load-once`?

Reply:
thx for the info, i'll align with Bamboo to add 'data-' prefix.

Reply:
No need to add 'data-' prefix:
'opf-load-once' is never added as attribute key, it is just a variable that decides if we add 'data-opf-resource' attribute or not.